code/datums/recipe.dm

```diff
@@ -39,6 +39,8 @@
 	var/result //Result of a complete recipe. result = /obj/item/weapon/reagent_containers/food/snacks/donut/normal
 	var/time = 10 SECONDS //Length of time it takes to complete the recipe. In 10ths of a second
 	var/priority = 0 //To check which recipe takes priority if they share ingredients
+	var/cookable_with = COOKABLE_WITH_ALL //How this recipe can be cooked, eg. COOKABLE_WITH_MICROWAVE (see setup.dm).
```

Comment:
Unsure this should be the default. Of course there are plenty of recipes that don't make sense in a microwave like ice cream sandwiches, but we don't know what's happening inside there, it could be space magic. Pan is a bit of a hard sell on a lot of recipes.

Reply:
That's true, I was thinking of differentiating the recipes a bit in a future PR. I figured doing it like this would be the least friction in terms of sentiment for now. I could set it to COOKABLE_WITH_MICROWAVE and then selectively make certain recipes that make more sense in a pan COOKABLE_WITH_ALL here if that's preferable though.

Reply:
I think it's better to start small and expand pan cooking rather than to take it away from some things later, which can be more contentious. Obviously our recipes page on the wiki will need some new method of showing if something is pan compatible.

Theoretically in the future we will get salad and soup assembles into bowls that have nothing to do with any form of heating element that won't overlap with either.

Reply:
`You begin tossing the contents of the salad bowl.`
Sounds nice.
I'll take a look at making some recipes not cookable in a pan. I guess for now anything that primarily requires a heating element like you said can be pan-cookable. I guess for the wiki the best thing would just be a "cooking vessel:" column or something.

Reply:
For now I kept the default as "ALL" sincemost of the recipes seem like they "could" be cooked in both. The main things I made microwave-only are sushi, ice cream, salad, and pickled stuff like sauerkraut.